modules/nw-rfhe-introduction.adoc

```diff
@@ -0,0 +1,71 @@
+
+=== REST APIs
+REST APIs provide a publish-subscribe model for the exhange of information between producing and subscribing applications. A REST API establishes the content required by the subscribing application (the call) and the content produced by the hardware event proxy (the response).
+
+=== Operator-managed pod
+The Operator makes use of custom resources to manage the pod containing the hardware event proxy and its components.
+
+=== Hardware event proxy
+A container created to handle Redfish events. At startup, the hardware event proxy queries the Redfish API and downloads all the Message Registries, including custom registries. The hardware event proxy then begins to receive subscribed events from the Redfish hardware.
+
+=== Cloud Native Event
+Cloud Native Events (CNE) is a Rest-API specification for defining the format of event data. https://github.com/redhat-cne/spec/blob/main/event_spec.md
+
+=== CNCF CloudEvents
+CloudEvents is a vendor-neutral specification for defining the format of event data.
+A specification developed by the Cloud Native Computing Foundation (CNCF) for describing event data in a common way.
+
+=== Dispatch router (AMQP 1.0 qpid)
+
+The dispatch router is responsible for the message delivery service between publisher and subscriber. AMQP 1.0 qpid is an open standard designed to support reliable, high-performance, fully-symmetrical messaging over the internet. AMQP connects systems, supports management processes with the information they need and reliably transmits onward the instructions that run a distributed network via a connection-oriented messaging API.
+
+=== Application pod (Subscriber)
+A virtual Distributed Unit for 5G and other EDGE applications.
+
+=== vDU
+Virtual distributed units within a network functions virtualization (NFV) infrastructure.
+
+NFV enables network functions such as monitoring, routing, firewalls, and load balancing to be packaged as virtual machines on any commodity hardware.
+
+=== Sidecar: Cloud event proxys
+A cloud events framework (based on ORAN API specifications) container image that provides an event publish-subscribe framework for hardware events.
+
+=== Application
+An application such as a vDU that consumes the messages coming from the cloud events framework. The response to these messages is determined by the customer application.
+
+[id="rfhe-data-flow_{context}"]
+== Redfish message parsing service
+
+In addition to relating Redfish events, the Redfish Hardware Event Proxy provides message parsing service for events without a `Message` property. The proxy downloads all the Redfish Message Registries including vendor specific registries from the hardware when it starts. If a event it received does not contain a Message property, the proxy will use the information from registries to construct the `Message` and `Resolution` properties and add them to the event before passing the event to the cloud events framework.
```

Comment:
The last sentence is a bit long, try dividing it up. Plus, avoid using "it". As someone who's not familiar with the tech, it's not clear what "it" refers to in this sentence.
> If an event **it** received does not contain a `Message` property, the proxy will use the information from registries to construct the `Message` and `Resolution` properties. Then, the proxy adds the properties to the event before passing the event to the cloud events framework.